SPHINXsys/src/shared/io_system/in_output.h

```diff
@@ -32,12 +32,9 @@
 #include "base_data_package.h"
 #include "sph_data_containers.h"
 #include "all_physical_dynamics.h"
+#include "parameterization.h"
 #include "xml_engine.h"
 
-#include "SimTKcommon.h"
-#include "SimTKmath.h"
-#include "Simbody.h"
```

Comment:
Ok. I put it back.

Reply:
I have checked and found that these includes are in base_data_type.h already.

Reply:
ok `base_data_type.h` seems to be a convenience header offering guaranteed include to SimTK. Otherwise the preferred practice is to avoid relying on transitive header include, unless it goes through catch-all convenience headers.